src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class LoginPage {
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )